packages/jest-parallel/src/__tests__/worker.test.js

```diff
@@ -0,0 +1,274 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @format
```

Comment:
I don't think these are needed in this code base

Reply:
😅 true! These ones come from a copy of an internal codebase. I will remove them 🙂